### PR TITLE
Add outreachy 2023-01 cfp blog post

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

## Docs PR Checklist

- [x] This PR is for a documentation page whose authoritative copy is in the opentelemetry.io repository.

### CODE REVIEW

**Review by svrnm — APPROVED** [review]

**Comment by svrnm** [comment]
overall lgtm, you might want to run prettier on it to have it styled properly.

One content suggestion, can you maybe point out what previous outreachy interns have created? This helps to understand what "medium" sized could mean.

TY

**Comment by jpkrohling** [comment]
PR updated with @svrnm's suggestions.

**Review by svrnm — APPROVED** [review]

File: content/en/blog/2023/outreachy-2023-01.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Call for participation - Outreachy May 2023
+linkTitle: End User Discussions Jan 2023
```
Comment (chalin) [inline content/en/blog/2023/outreachy-2023-01.md:3]:
```suggestion
linkTitle: Outreachy CFP '23
```

File: content/en/blog/2023/outreachy-2023-01.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+and more.
+
+Here's where you can get more information about Outreachy and our participation
+on previous cohorts:
+
+- [Outreachy](https://www.outreachy.org/)
+- [CNCF OpenTelemetry: Past community participation in Outreachy](https://www.outreachy.org/communities/cfp/cncf-tracing/)
+
+The easiest way to get in touch with us if you have a project idea or want to be
+a mentor is via CNCF Slack, #outreachy.
```
Comment (chalin) [inline content/en/blog/2023/outreachy-2023-01.md:45]: Add link to CNCF Slack?

**Review by chalin — COMMENTED** [review]
See inline comments.

I'd also suggest renaming the file from:

- `content/en/blog/2023/outreachy-2023-01.md`

to

- `content/en/blog/2023/outreachy-cfp.md` since `2023` is already in the path.

File: content/en/blog/2023/outreachy-2023-01.md (hunk 1 above)
Comment (jpkrohling) [reply content/en/blog/2023/outreachy-2023-01.md]: Accepted, I just spelled out CFP as 'call for participation', as CFP can also be read as 'call for papers'.

**Comment by jpkrohling** [comment]
Thanks, the PR is now updated. I decided to name the file differently, as there are two Outreachy cohorts each year, and this is related to May's.

**Review by chalin — APPROVED** [review]

**Comment by svrnm** [comment]
🎉

**Pull request merged**